test/lock-parser/fixtures/additional-global-options/out.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dependencies": [
```

Comment:
```suggestion
  "importTargets: false"
  "farmework": "net452"
  "repositories": {
  "remote": "https://api.nuget.org/v3/index.json"
  "name": "nuget"
  }
  }
```

Reply:
something like this should be very achievable in our current setup.